src/main/java/tdl/s3/helpers/FileHelper.java

```diff
@@ -5,7 +5,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 
-public class FileHelper {
+public final class FileHelper {
```

Comment:
To test utility classes have a look at this: https://github.com/Deployable-code/osgb36-letter-conversion/blob/master/src/test/java/ro/ghionoiu/osgb/letters/util/GridMathTest.java